vm_scripts/networks.py

```
@@ -0,0 +1,594 @@
+    # not being used at this moment
+    # this function will be used in future version
+    def ip_assign(self, service_instance, vm_name,host_name, ip, default_values,type,username=None, password=None):
+        try:
+            
+            content = service_instance.RetrieveContent()
+            vm = self._vm_helper.get_obj(content, [vim.VirtualMachine], vm_name)
+
+            if "folder" in default_values:
+                vm_folder = default_values['folder']
+            else:
+                vm_folder = None
```

Ruff (0.8.2)

Use ternary operator `vm_folder = default_values['folder'] if 'folder' in default_values else None` instead of `if`-`else`-block

Replace `if`-`else`-block with `vm_folder = default_values['folder'] if 'folder' in default_values else None`

(SIM108)